src/lib/core/CHIPError.h

```diff
@@ -1891,12 +1891,12 @@ using CHIP_ERROR = ::chip::ChipError;
 #define CHIP_ERROR_INCOMPATIBLE_SCHEMA_VERSION                 CHIP_CORE_ERROR(0xa3)
 
 /**
- *  @def CHIP_ERROR_MISMATCH_UPDATE_REQUIRED_VERSION
+ *  @def CHIP_ERROR_MISMATCH_WRITE_REQUIRED_VERSION
  *
  *  @brief
- *    Encountered a mismatch between update required version and current version
+ *    Encountered a mismatch between update write version and current version
  */
-#define CHIP_ERROR_MISMATCH_UPDATE_REQUIRED_VERSION            CHIP_CORE_ERROR(0xa4)
+#define CHIP_ERROR_MISMATCH_WRITE_REQUIRED_VERSION            CHIP_CORE_ERROR(0xa4)
```

Comment:
Oh, and this is unused, right?

Reply:
reverted it back, thanks

Reply:
The point is, can we just remove it?

Reply:
since this is middle one, if I remove this one, then I have to shift all error number in this PR...need to figure out a better way to removed unused error...

Reply:
We could remove without renumbering.  But yes, separate PR or this.

Reply:
thanks, would remove it without renumbering now.